solidity/test/TestTokenStake.js

```diff
@@ -562,7 +565,7 @@ contract('TokenStaking', function(accounts) {
         operatorOne, operatorContract, {from: authorizer}
       )
   
-      await mineBlocks(initializationPeriod - 1)
+      await increaseTime(initializationPeriod - 1)
```

Comment:
I think this one should be `increaseTime(initializationPeriod)` to cover the corner case.